components/prism-fsharp.js

```diff
@@ -0,0 +1,29 @@
+Prism.languages.fsharp = Prism.languages.extend('clike', {
+	'comment': [
+		{
+			pattern: /(^|[^\\])\(\*[\w\W]*?\*\)/g,
+			lookbehind: true
+		},
+		{
+			pattern: /(^|[^\\:])\/\/.*/g,
+			lookbehind: true
+		}
+	],
+	'keyword': /\b(abstract|and|as|assert|base|begin|class|default|delegate|do|done|downcast|downto|elif|else|end|exception|extern|false|finally|for|fun|function|global|if|in|inherit|inline|interface|internal|lazy|let|let!|match|member|module|mutable|namespace|new|not|null|of|open|or|override|private|public|rec|return|return!|select|static|struct|then|to|true|try|type|upcast|use|use!|val|void|when|while|with|yield|yield!|asr|land|lor|lsl|lsr|lxor|mod|sig|atomic|break|checked|component|const|constraint|constructor|continue|eager|event|external|fixed|functor|include|method|mixin|object|parallel|process|protected|pure|sealed|tailcall|trait|virtual|volatile)\b/g,
+	'string': /@?("""|"|')((\\|\n)?.)*?\1B?/g,
+	'preprocessor': /^\s*#.*/gm,
+	'number': [
+		{
+			pattern: /\b-?(0x[\da-fA-F]*.(un|LF)?)\b/g
```

Comment:
Why the dot? Is it supposed to be an actual dot or any character? I don't get this.
Also, I think you should use `+` instead of `*` since we want at least one char.
There are still unneeded parentheses around the whole number.
Finally, [this page on the MSDN](https://msdn.microsoft.com/en-us/library/dd233193.aspx) show an example using `lf` in lowercase, that is not handled here.